index.html

```diff
@@ -13,11 +13,11 @@
 
 <body>
   <div class="content-wrapper">
-    <nav>
-      <a href='https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97' class="btn-login">
-        Login
-      </a>
-    </nav>
+    <div id="user-login">
```

Comment:
Why change the `nav` into a `div`?
`nav` is a more semantic tag for html.

Read more here: https://css-tricks.com/why-how-and-when-to-use-semantic-html-and-aria/

Reply:
Cool. Will keep this mind. The article is nice :)